Makefile

```diff
@@ -101,3 +105,4 @@ ifndef HAS_GOX
 	go get -u github.com/mitchellh/gox
 endif
 	glide install
+	@echo Please run pip install -r expansion/requirements.txt. You might need sudo.
```

Comment:
Will this muck with virtual environments or anything, @mboersma ? Any chance that this will be a nasty surprise to the user?